config/ext_jenkins-weekly.yaml

```diff
@@ -0,0 +1,176 @@
+serviceAccount:
+  create: true
+  name: jenkins-controller
+serviceAccountAgent:
+  create: true
+  name: jenkins-agent
```

Comment:
```suggestion
serviceAccountAgent:
  create: false
```

I would avoid creating an unused service account (we don't plan to use kubernetes agents on this one, or at least not in this cluster)